src/Core/src/Platform/Windows/NavigationViewExtensions.cs

```diff
@@ -270,5 +272,47 @@ public static void UpdateFlyoutWidth(this MauiNavigationView navigationView, IFl
 			// At some point this Template Setting is going to show up with a bump to winui
 			//handler.PlatformView.OpenPaneLength = handler.PlatformView.TemplateSettings.OpenPaneWidth;
 		}
+
+		public static async Task UpdateFlyoutIconAsync(this MauiNavigationView navigationView, IImageSource? imageSource, IImageSourceServiceProvider? provider)
+		{
+			var togglePaneButton = navigationView.TogglePaneButton;
+
+			if (togglePaneButton is null)
+				return;
+
+			var animatedIcon = togglePaneButton.GetFirstDescendant<AnimatedIcon>();
+			
+			if (animatedIcon is null)
```

Comment:
I'm running this locally and this is always null for me, and the icon isn't changing.

I tested on the sandbox

```XAML
<Shell
    xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
    xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
    x:Class="Maui.Controls.Sample.SandboxShell"
    xmlns:local="clr-namespace:Maui.Controls.Sample"
    x:Name="shell"
    FlyoutIcon="dotnet_bot.png">
```

Reply:
Have done some tests and I think have been an error doing fallback to the original icon. Fixed here https://github.com/dotnet/maui/pull/14663/commits/7766d4c9c356ac6542336b27566ef049bb9ef85f

Tested on the Gallery and with Device tests.